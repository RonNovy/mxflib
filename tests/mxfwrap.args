-k=64 -a -f -r25/1 ../../small.wav ../../mxfwrap_wav.mxf
