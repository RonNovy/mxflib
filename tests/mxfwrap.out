MXFlib File Wrapper

KAGSize     = 64

Input file  = ../../small.wav
Output file = ../../mxfwrap_wav.mxf

Output OP = OP-Atom
An updated header will be written after writing the footer
Header padding has been increased to the empirical minimum: 16384 bytes
At least 16384 padding bytes will be left after the initial writing of the header
Index tables will be written for each frame wrapped essence container
Index table segments will not share a partition with essence
ERROR: Could not identify the essence in file "../../small.wav"
